lib/task_2/task2.dart

```diff
@@ -0,0 +1,35 @@
+// Реализуйте методы для преобразования целых чисел
+// из десятичной системы в двоичную и обратно.
+
+int convertBinaryToDecimal(int binaryNumber) {
```

Comment:
```suggestion
int convertBinaryToDecimal(String binaryNumber) {
```

Всё-таки в параметрах не должны поступать десятеричные числа. Пусть будет строка, которая представляет собой запись двоичного числа.